Fastfile

```diff
@@ -164,14 +157,24 @@ def _get_team_id()
   team_id
 end
 
-def _parse_provision_file()
-  sh 'RUBYSCRIPT="$(echo \'def target(targetName, &doBlock)
-    profileName = doBlock.call
-    print targetName + "_" + "PROVISIONING_PROFILE = " + profileName + "\n"
+def _create_xcconfig(filename)
+  # TODO: Replace spaces with underscores?
+  # TODO: Try not to need `TAB_PROJECT_PATH`
+  project = Xcodeproj::Project.open(ENV['TAB_PROJECT_PATH'])
+  project.targets.each do |target|
+    profile = _get_profile_for_target(target)
+    if !profile.nil?
+      sh "echo \"#{target.name}_PROFILE_SPECIFIER=#{profile}\" >> #{filename}"
+    end
   end
-  \' & cat ProvFile)"
-  echo "${RUBYSCRIPT}" | ruby > TAB.release.xcconfig
-  echo \'PROVISIONING_PROFILE_SPECIFIER=$($(TARGET_NAME)_PROVISIONING_PROFILE)\' >> TAB.release.xcconfig'
+  sh "echo 'PROVISIONING_PROFILE_SPECIFIER=$($(TARGET_NAME)_PROFILE_SPECIFIER)' >> #{filename}"
```

Comment:
Might be nicer to do the file writing in Ruby. Do something like gather each line for the file in an array, then when you have them all call
```ruby
File.write(filename, lines.join("\n"))
```

Reply:
That would also fix the appending problem

Reply:
Ooh nice I'll have a look into that

Reply:
DELICOUS. Just running the lane now to double check